themes/hugoplate/layouts/partials/components/sponsor-card.html

```diff
@@ -0,0 +1,37 @@
+<div class="rounded-lg border-solid border p-4">
+  {{ $image:= .Params.image }}
+  {{ if $image }}
+    {{ partial "image" (dict "Src" $image "Alt" .Title "Class" "mb-6 w-full rounded") }}
```

Comment:
Ik zou hier eerder de plaatjes in de content folder zetten: https://gohugo.io/content-management/page-resources/

Reply:
De content folder blijkt niet te doen. Ik heb de next best thing gedaan: wel in de global assets maar de images inladen op basis van bestandsnaam